#ifndef AR_TAG_PAD_H
#define AR_TAG_PAD_H

#include <iostream>
#include <ros/ros.h>
#include <pthread.h>
#include <string>
#include <cv_bridge/cv_bridge.h>
#include <tf/transform_listener.h>
#include <tf/transform_broadcaster.h>
#include <sensor_msgs/image_encodings.h>

#include <opencv2/core/core.hpp>
#include <opencv2/imgproc/imgproc.hpp>
#include <opencv2/highgui/highgui.hpp>
#include <opencv2/imgproc/imgproc_c.h>
#include <opencv2/features2d/features2d.hpp>
#include <opencv2/calib3d/calib3d.hpp>

//#include "../include/global_variable.h"


#endif // AR_TAG_PAD_H
